include/CppUTest/CppUTestConfig.h

```diff
@@ -199,6 +199,27 @@
 #define CPPUTEST_CHAR_BIT 8
 #endif
 
+/* Support for "long long" type */
+#ifdef CPPUTEST_USE_LONG_LONG
```

Comment:
Ok. Made 3 pull requests related to packaging. Now it will install the config.h but renamed and prefixed, so that it shouldn't lead to conflict. I'm not using it in the CppuTestConfig.h, but that is the next step. It turned out easier than I had expected (good).

The same changes will need to be made on the CMake side, I guess. Delivering CppUTest with an empty CppUtestGeneratedConfig.h ought to be able to deal with the others, such as VC++.